pkg/sbom/sbom.go

```diff
@@ -107,6 +112,20 @@ func Generate(ctx context.Context, inputFilePath string, f io.Reader, distroID s
 	packageCollection := createdSBOM.Artifacts.Packages
 	packageCollection.Add(*apkPackage)
 
+	if cfg.Relationships.ExcludeBinaryPackagesWithFileOwnershipOverlap {
```

Comment:
I've found a bit more context of this flag in here: https://github.com/anchore/syft/issues/931. If anyone is reviewing this PR.